### PR TITLE
chore: update to see webhook creation error handling to see error in prod

### PR DESCRIPTION
**Pull request by sean-brydon**
Pr adds a log so we can see the error that happens here in prod. 

response doesnt throw if response.ok is false -> This error was never being hit.

### CODE REVIEW

File: packages/app-store/paypal/lib/Paypal.ts [hunk 1]
```diff
@@ -191,17 +191,16 @@ class Paypal {
       ],
     };
 
-    try {
```
Comment (CarinaWolli) [inline packages/app-store/paypal/lib/Paypal.ts:194]: can we keep the try/catch here?

File: packages/app-store/paypal/lib/Paypal.ts [hunk 2]
```diff
@@ -197,12 +197,14 @@ class Paypal {
...
+      if (!response.ok) {
+        throw new Error(await response.json());
       }
-    } catch (error) {
-      console.error(error);
+
+      const result = await response.json();
+      return result.id as string;
+    } catch (e) {
+      console.error(e);
```
Comment (Udit-takkar) [inline packages/app-store/paypal/lib/Paypal.ts:207]: can't we use logger from /lib/logger?

Comment (sean-brydon) [reply packages/app-store/paypal/lib/Paypal.ts:207]: Good shout

**Review by Udit-takkar — DISMISSED** [review]

File: packages/app-store/paypal/lib/Paypal.ts [hunk 3]
```diff
@@ -197,12 +198,14 @@ class Paypal {
         body: JSON.stringify(body),
       });
 
-      if (response.ok) {
-        const result = await response.json();
-        return result.id as string;
+      if (!response.ok) {
+        throw new Error(await response.json());
```
Comment (CarinaWolli) [inline packages/app-store/paypal/lib/Paypal.ts:202]: Should we add a message string in here, something like: 
```
const message = `${response.statusText}: ${JSON.stringify(await response.json())}`;
throw new Error(message);
```

Comment (Udit-takkar) [reply packages/app-store/paypal/lib/Paypal.ts]: Done

**Review by CarinaWolli — APPROVED** [review]

**Pull request merged**